tests/unit/lms/services/user_test.py

```diff
@@ -164,7 +164,7 @@ def test_get_users_for_course(
         query = service.get_users_for_course(
             role_scope=RoleScope.COURSE,
             role_type=RoleType.LEARNER,
-            course_id=course.id,
+            lms_course=course.lms_course,
```

Comment:
The test don't need any further changes so that's a good sign that the queries are equivalent.